napari_clusters_plotter/_utilities.py

```diff
@@ -235,7 +235,9 @@ def generate_cluster_image(label_image, label_list, predictionlist):
     predictionlist_new = np.array(predictionlist) + 1
     label_list = np.array(label_list)
 
-    return map_array(label_image, label_list, predictionlist_new).astype("uint64")
+    return map_array(np.array(label_image), label_list, predictionlist_new).astype(
```

Comment:
```suggestion
    return map_array(np.asarray(label_image), label_list, predictionlist_new).astype(
```

Reply:
Ok, changed it

Reply:
Thanks! Otherwise it was trying to make a copy even if data is already a numpy array and running out of memory on my laptop

Reply:
yes, important change :-)